cluster/external/provider.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+function _kubectl(){
```

Comment:
Any particular reason you didn't modify cluster/kubectl.sh to check for external provider and point it to the right kubectl in that case?

Reply:
That's another approach (maybe better, not sure); I went this route because it was easy to just convert to a function and stuff uniqueness into the appropriate provider script and keep them extendable and independent.  I can look at the other possibility though and see if it works better.  First to figure out what I broke :)